### DISCUSSION TITLE
Adding Landsat Thermal band (bands 6/10) to the Datacube

### DISCUSSION THREAD

**Discussion by stukrause** [davidfrantz/force#267] [Q&A] [answered]
Hello, we have FORCE working well for all of Germany with Sentinel 2 and Landsat. Unfortunately the FORCE original workflow leaves out the thermal band for the output. Does anyone have any tips in how we could modify the workflow to add the uncalibrated thermal band (Level I) to the output?

Thanks!

**Comment by davidfrantz** [comment, marked as answer]
Hi @stukrause,

currently, there is no easy way to achieve this and a solution would require some coding.

If you are keen on giving it a try, ``src/lower-level/atmo-ll.c`` is the file to look at.

I think that this needs at least one new function (similar to ``compile_l2_hot``), which should be called in ``compile_level2`` after ``compile_l2_hot``, plus some additional stuff before like in the *hot* case.

Cheers,
David

**Comment by stukrause** [comment]
Thanks @davidfrantz for your quick reply! I see what I can do...
